llvm/lib/DWARFLinker/Parallel/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ add_llvm_component_library(LLVMDWARFLinkerParallel
   SyntheticTypeNameBuilder.cpp
 
   ADDITIONAL_HEADER_DIRS
-  ${LLVM_MAIN_INCLUDE_DIR}/llvm/DWARFLinkerParallel
+  ${LLVM_MAIN_INCLUDE_DIR}/llvm/DWARFLinker/Parallel
```

Comment:
same comment – since things worked with the old, wrong, name, maybe we can toss it altogether? (but seems like something for a different pr)